packages/ui/src/visually-hidden/style.module.css

```
@@ -0,0 +1,17 @@
+@layer wp-ui-utilities, wp-ui-components, wp-ui-compositions, wp-ui-overrides;
+
+@layer wp-ui-components {
+	.visually-hidden {
+		border: 0;
+		clip: rect(1px, 1px, 1px, 1px);
+		--webkit-clip-path: inset(50%);
```

```suggestion
		-webkit-clip-path: inset(50%);
```

I think there's a typo here suggesting this is a CSS var instead of a vendor prefix. Also, is this actually needed?

---

Good point, removed in 58b71b3.